src/tools/compiletest/src/lib.rs

```diff
@@ -716,7 +718,7 @@ fn make_test(
     inputs: &Stamp,
     poisoned: &mut bool,
 ) -> Vec<test::TestDescAndFn> {
-    let test_path = if config.mode == Mode::RunMake {
+    let test_path = if config.mode == Mode::RunMake && !testpaths.file.join("__rmake.rs").exists() {
```

Comment:
Is there a reason for __rmake being named with the underscores?

Reply:
Yes -- I didn't know what to call them so `__rmake` is a placeholder name 😄

Reply:
I'd go with something like rmake.rs - I don't think we need anything too special.